dot/core/service.go

```diff
@@ -249,6 +250,13 @@ func (s *Service) handleBlock(block *types.Block, state *rtstorage.TrieState) er
 }
 
 func (s *Service) handleCodeSubstitution(hash common.Hash, state *rtstorage.TrieState) error {
+	return s._handleCodeSubstitution(hash, state, wasmer.NewInstance)
```

Comment:
whats the point of this? also `_` prefixed funcs are not really a thing in go afaik

Reply:
oh its so you can inject the function during tests i guess?

Reply:
yup yup exactly! Makes it so I dont have to generate a valid wasm instance for the mocked unit tests so it goes a lot faster (no more need to generate wasm files)